casepropods/family_connect_subscription/plugin.py

```diff
@@ -57,19 +65,45 @@ def read_data(self, params):
                 "value": subscription['next_sequence_number']})
             # Add the schedule
             schedule_id = subscription['schedule']
-            schedule = stage_based_messaging_api.get_schedule(schedule_id)
+            schedule = self.stage_based_messaging_api.get_schedule(schedule_id)
             subscription_data['rows'].append({
                 "name": "Schedule",
                 "value": self.format_schedule(schedule)})
             # Add the active flag
             subscription_data['rows'].append({
                 "name": "Active",
                 "value": subscription['active']})
+            if subscription['active']:
+                active_sub_ids.append(subscription['id'])
             # Add the completed flag
             subscription_data['rows'].append({
                 "name": "Completed",
                 "value": subscription['completed']})
             content['items'].append(subscription_data)
+
+        actions = [{
+            'type': 'full_opt_out',
+            'name': 'Full Opt-Out',
+            'confirm': True,
+            'busy_text': 'Processing...',
+            'payload': {
+                'contact_id': case.contact.id,
+                'subscription_ids': active_sub_ids
+            }
+        }]
```

Comment:
@rudigiesler currently that would require us to retrieve the identity and check their addresses to see if they've opted out. I'm not sure how we would handle cases where they've opted out some addresses but not others.